tests/Commands/UserModelGeneratorTest.php

```diff
@@ -60,14 +60,13 @@ public function testGenerateUserModel(): void
 
     public function testGenerateUserModelCustomNamespace(): void
     {
-        command('shield:model MyUserModel --namespace Tests');
-        $filepath = Tests . 'MyUserModel.php';
-
+        command('shield:model MyUserModel --namespace CodeIgniter\\\\Shield');
+        $filepath = VENDORPATH . 'codeigniter4/shield/src/Models/MyUserModel.php';
```

Comment:
Any personal help here :upside_down_face:

![Screenshot 2022-11-02 091006](https://user-images.githubusercontent.com/9530214/199408976-a7b645c9-da76-429c-b4a9-4644e368129e.jpg)

Reply:
Should the path be `ROOTPATH . 'src/Models/MyUserModel.php'`?

Reply:
Hm okay. Use `HOMEPATH` instead of `ROOTPATH`?

Reply:
The `ROOTPATH` equals`/home/runner/work/shield/shield/vendor/codeigniter4/framework/`, which is not correct .

```
There was 1 failure:

1) Tests\Commands\UserModelGeneratorTest::testGenerateUserModelCustomNamespace
Failed asserting that file "/home/runner/work/shield/shield/vendor/codeigniter4/framework/src/Models/MyUserModel.php" exists.

/home/runner/work/shield/shield/tests/Commands/UserModelGeneratorTest.php:67
phpvfscomposer:///home/runner/work/shield/shield/vendor/phpunit/phpunit/phpunit:97

FAILURES!
Tests: 206, Assertions: [52](https://github.com/codeigniter4/shield/actions/runs/3375319598/jobs/5601830888#step:11:53)4, Failures: 1.
```

Reply:
https://github.com/codeigniter4/shield/blob/9eaa72ae6b4e2fee04219e56bb93e694a315f800/phpunit.xml.dist#L84

Reply:
Thank you so much @paulbalandan, it finally worked. :relaxed: